packages/scoutgame/src/builders/getBuilderScouts.ts

```diff
@@ -38,12 +40,15 @@ export async function getBuilderScouts(builderId: string): Promise<BuilderScouts
     }
   });
 
+  // TODO: Figure out why there are some events that don't have a scoutWallet
+  const validPurchaseEvents = nftPurchaseEvents.filter((e) => e.scoutWallet);
```

Comment:
i would check to see if they have a senderWallet address? These may be 'burned' nft purchases. If we send them to NULL address, there is no scout wallet

Reply:
added a check to make sure we don't fetch burned nfts